### PR TITLE
qeu: buildroot: always enable networking

### PR DESCRIPTION
**Pull request by jforissier**
Signed-off-by: Jerome Forissier <jerome.forissier@linaro.org>

### CODE REVIEW

File: docs/qemu.md [hunk 1]
```diff
@@ -45,12 +45,5 @@ To mount host folder in QEMU, simply run:
 ```bash
 $ mount -t 9p -o trans=virtio host <mount_point>
 ```
-# 5. SLiRP user networking
-To enable SLiRP user networking just set `QEMU_USERNET_ENABLE ?= y` in `common.mk`.
-After booting QEMU VM, eth0 will automatically receive IP address via DHCP
-
-*Important* Take into account that ICMP doesn't work in SLiRP mode,
```
Comment (jenswi-linaro) [inline docs/qemu.md]: Is this note still relevant?

Comment (jforissier) [reply docs/qemu.md]: It is. Well, `ping localhost` works, but pinging external hosts does not.

**Comment by jforissier** [comment]
@jenswi-linaro almost no additional delay to boot with this patch. IP configuration takes no more than 1 second.

**Comment by jenswi-linaro** [comment]
Please keep the description of networking. With that:
`Reviewed-by: Jens Wiklander <jens.wiklander@linaro.org>`

**Comment by jforissier** [comment]
@jenswi-linaro I have slightly reworded the doc and applied your R-b. Thanks.

**Pull request merged**